fili-core/src/main/java/com/yahoo/bard/webservice/table/BasePhysicalTable.java

```diff
@@ -93,20 +108,13 @@ public String getPhysicalColumnName(String logicalName) {
     }
 
     /**
-     * Update the working intervals with values from a map.
+     * Used only for testing to inject test availability data into table.
      *
-     * @param segmentMetadata  A map of names of metrics and sets of intervals over which they are valid
-     * @param dimensionDictionary  The dimension dictionary from which to look up dimensions by name
+     * @param availability  The test availability for this table
+     *
+     * @deprecated  Should avoid this method and refine testing strategy to remove this method
      */
-    public void resetColumns(SegmentMetadata segmentMetadata, DimensionDictionary dimensionDictionary) {
-        setAvailability(new ImmutableAvailability(
-                name.asName(),
-                segmentMetadata.getDimensionIntervals(),
-                segmentMetadata.getMetricIntervals(),
-                dimensionDictionary
-        ));
-    }
-
+    @Deprecated
     protected void setAvailability(Availability availability) {
```

Comment:
As long as AvailabilityTestingUtils is a Groovy class, it should be able to see it. IntelliJ may _warn_ you (or pretend that it can't see it), but Groovy ignores it.